data/pug/pug news.txt

```diff
@@ -352,9 +352,8 @@ news "worker on pug-occupied planet after fw"
 
 news "pug on pug planets during fw"
 	location
-		government Pug
-		near Deneb
-		near Vega
+		government "Pug"
```

Comment:
```suggestion
		government "Pug"
		near Deneb
 		near Vega
```
Why remove those filters?

Reply:
Because they're redundant. There are never any `"Pug"` planets that are not neighbouring those two systems. (Wanderer Pug use a different government.)

Reply:
OK